UsingTheServer.md

```diff
@@ -22,54 +22,127 @@ repositories {
 }
 ```
 
-## Using the server
+## Using the Server
 
 Usage of the server requires some setup within your application.
 
-Your application should call `HttpServer start` in order to run the server. Below you will find documentation for `HttpServer` and other classes required for setup.
+Begin by creating an instance of the HttpServer (see [API](#api) below for `Logger` information): `HttpServer(Logger logger)`. From here, you can being configuring your server.
+
+### `.addStatic(Path root)`
+
+`addStatic` sets up built-in middleware for serving static files.  The `root` parameter specifics the root directory from which static resources will be served.  If a resource matching the requested resource is not found in the root directory, the request passes to the router to check for matching routes before returning a 404.
+
+Static middleware will not be included if `addStatic` is not set.
+
+### `.router(Routes routes)` 
+
+`router` creates an instance of the server router which uses the provided routes. The router is the last layer for matching request resources.  If a match is found for the requested resource, the router will use the controller specified in `routes` (see [Setting Up Routes](#setting-up-routes)). If the router does not find a route matching the requested resource, it will return a 404.
+
+A route-less router will be used if `router` is not set.
+
+### `.start(int port)`
+
+Once the above methods are used to configure the server, use `start` with a valid port number to start the server.
+
+## Example Setup
+
+Below is a simple example of how server setup might look:
+
+```java
+new HttpServer(logger)
```

Comment:
Is this supposed to be `new Application(logger)` now instead?

Reply:
Sure is.  Thank you.  Just when I thought I got all the ReadMe stuff.